haskell/private/actions/compile.bzl

```diff
@@ -105,6 +113,30 @@ def _process_hsc_file(hs, cc, hsc_flags, hsc_inputs, hsc_file):
 
     return hs_out, idir
 
+# Not strictly for headers but that's my motivating example..
+# TODO[GL} this will fail if the header is outside of our projec root
```

Comment:
Not sure what to do about this - my judgement is to just "leave it as future work", unless there's some straightforward solution I'm missing..